.jshintignore

```diff
@@ -0,0 +1,6 @@
+src/libs/potrace.js
```

Comment:
The `Potrace` version we use is different from the one in NPM that's why we don't use it.